db/migrate/20160513124246_remove_features_from_marketplace_trials.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveFeaturesFromMarketplaceTrials < ActiveRecord::Migration
+  def change
+    remove_column :marketplace_trials, :features, :text, after: :community_id
```

Comment:
The "Add features" and "Remove features" migrations could be removed. (and also the member limit)